src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.cs

```diff
@@ -156,6 +156,8 @@ public sealed partial class JsonContent : System.Net.Http.HttpContent
         [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("JSON serialization and deserialization might require types that cannot be statically analyzed. Use the overload that takes a JsonTypeInfo or JsonSerializerContext, or make sure all of the required types are preserved.")]
         [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("JSON serialization and deserialization might require types that cannot be statically analyzed. Use the overload that takes a JsonTypeInfo or JsonSerializerContext.")]
         public static System.Net.Http.Json.JsonContent Create<T>(T inputValue, System.Net.Http.Headers.MediaTypeHeaderValue? mediaType = null, System.Text.Json.JsonSerializerOptions? options = null) { throw null; }
+        public static System.Net.Http.Json.JsonContent Create<TValue>(TValue? inputValue, System.Text.Json.Serialization.Metadata.JsonTypeInfo<TValue> jsonTypeInfo, System.Net.Http.Headers.MediaTypeHeaderValue? mediaType = null)  { throw null; }
```

Comment:
Would it be possible to match the type parameter name with that of the existing overload?

Reply:
Done